packages/vaadin-grid/src/vaadin-grid-keyboard-navigation-mixin.js

```diff
@@ -719,7 +722,7 @@ export const KeyboardNavigationMixin = (superClass) =>
     /**
      * Fired when a cell is focused with click or keyboard navigation.
      *
-     * Use {@see GridElement#getEventContext} to get detailed information about the event.
+     * Use context property of @type {GridCellFocusEvent} to get detail information about the event.
```

Comment:
I think `@type {GridCellFocusEvent}` is not correctly applied here. Seems like you use it to define the type of a variable or property. In this case we just want to link/reference to the relevant type. So `@see` or `@link` would be more appropriate I think.

Note that our current docs tooling doesn't actually process these tags, but it still makes sense to maintain them if/when we switch to different tooling.